stepDefinitions/clearPage.js

```diff
@@ -1,7 +1,13 @@
-"use strict";
-const search = require("../pages/searchPage");
-const { Then } = require("cucumber");
+import GoogleSearch from '../pages/GoogleSearch';
+import { defineSupportCode } from "cucumber";
+
+// GoogleSearch page instance
+const googleSearch = new GoogleSearch();
+
+defineSupportCode(({ Then }) => {
```

Comment:
defineSupportCode  class is no more required to be imported, since cucumber 2.3.0 they have added aliases for the API methods , pls see the [changelog](https://github.com/cucumber/cucumber-js/blob/master/CHANGELOG.md#230-2017-06-01). It helps in better readability and users switching from other languages would find it easy to digest!

Reply:
this detail went unnoticed.... rsrs